documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -355,7 +355,7 @@ The documentation template below must be used as part of the API documentation i
 ```
 ### Authorization and authentication
 
-[Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md) provides details on how a client requests an access token.
+The "Camara Security and Interoperability Profile" provides details on how a client requests an access token. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of the Profile.
```

Comment:
I guess that the link is not formatted as `[](url)` on purpose, right?

Reply:
Actually I hadn't thought about it, but yes, it makes sense to have the URL openly visible.

Reply:
looks good (OK from Release Mgmt if Herbert did not already have that hat)